### ISSUE TITLE
Reflection Probe Bounding / Range Is Wrong

### ISSUE THREAD

**Issue by Enverex** [Frooxius/NeosPublic#1389]
Sorry for the presumptuous title but I wasn't sure how else to word it. Reflection Probes appear to either cover more area than they should, or their origin point(s) are offset. I've been trying to figure out why walls that aren't in the range of a probe were showing the wrong probe's reflection. They were in the range of probe A, near (but not inside) probe B's range. Yet they should show probe B's reflection.

Here's a good example of this. The room at the bottom of the screen has a reflection probe covering the whole room (and no further). A second reflection probe is currently selected in the room above and the boundary can be seen by the gizmo. I've put a reflective ball in the bottom left of the screen. As you can see, it's showing the reflection of the room above which it's well outside of, and not the reflection of the room who's probe it's well within.

![image](https://user-images.githubusercontent.com/958670/100141827-e5298480-2e8a-11eb-869a-521c16b7c952.png)

In the same picture you can also see the wall the ball is next to showing the wrong reflection, as well as the checker wall on the right showing the wrong reflection too (it's showing the reflection for the corridor behind it, not the room it's in).

It's not a scale issue as all the slots leading up to (and the probe itself) are all 1,1,1 scale.

**Comment by InsaneGrox** [comment]
This might be by design, the blending range in neos appears to act as additional range for the reflection probe where it smoothly falls off, where the blend distance goes outwards instead of inwards.

**Comment by Enverex** [comment]
It's well outside of the blend area which is actually a very small range. Messing with the importance isn't an option because that, paired with the weird boundaries leads to an impossible game of whack-a-mole with almost everywhere then ending up with the wrong reflections.

**Comment by InsaneGrox** [comment]
The blend range is in meters, so even though the value is 1.00, that is 1 meter, so the range is actually quite big, again, I think it's by design, however a potential solution would be showing the outer bounds as a second bounding box to prevent confusion surrounding this.
TLDR: 1 meter blending area is not "very small range", it's a whole extra meter in every direction.

**Comment by Enverex** [comment]
Lowering the blend area to 0.1 (or even 0) doesn't result in any change in behaviour, so it's not that.

**Comment by Frooxius** [comment]
Should be fixed in 2020.11.25.1148, thanks for the report! The bounds for the probes are in global space, but this was showing them in local. To show the bounds you have to enable the "ShowDebugVisual" property.
